src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -536,6 +541,8 @@ def launch_fluent(
             if mode != LaunchModes.SOLVER_ICING:
                 env["APP_LAUNCHED_FROM_CLIENT"] = "1"  # disables flserver datamodel
             kwargs = _get_subprocess_kwargs_for_fluent(env)
+            if cwd:
```

Comment:
@abhishekchitwar @mkundu1 is there a reason we can't simply add this to the json file?

Reply:
@seanpearsonuk, it seems JSON has all options related to Fluent specific options (as similar to Fluent command line / batch arguments). However, "cwd" is subprocess specific argument.

Reply:
@abhishekchitwar sorry, I didn't look properly before commenting, thanks.